apps/docs-snippets/src/guide/introduction/getting-started.test.ts

```diff
@@ -1,19 +1,14 @@
-import {
-  FUEL_NETWORK_URL,
-  FUEL_TESTNET_NETWORK_URL,
-  Provider,
-  Wallet,
-  WalletUnlocked,
-} from 'fuels';
+import { FUEL_NETWORK_URL, TESTNET_NETWORK_URL, Provider, Wallet, WalletUnlocked } from 'fuels';
```

Comment:
Do we still need `FUEL_NETWORKL_URL`? 🤔 

cc @Torres-ssf

Reply:
@arboleya the `FUEL_NETWORK_URL` constant is mainly ( if not only ) used within our tests, and its value is: `http://127.0.0.1:4000/v1/graphql`. 

@FuelLabs/sdk-ts Does anyone remember why we export this constant from our umbrella package?

Reply:
@Torres-ssf I was thinking if we could go with something like this:
 - `LOCAL_NETWORK_URL`
 - `DEVNET_NETWORK_URL`
 - `TESTNET_NETWORK_URL`
 - `NETWORK_URL` (for upcoming mainnet)

Thoughts?

Reply:
@arboleya This is a good suggestion. So the `LOCAL_NETWORK_URL` will be `http://127.0.0.1:4000/v1/graphql` ?

Reply:
The one caveat is that when copying from snippets, `FUEL_NETWORK_URL` used a ternary that could evaluate but live and local networks, so it could be extracted from snippets. Whereas someone now will need to change the constant dependent on if they are working on testnet or locally.

Reply:
Good point. Do you think this is a problem tho? Or a feature?

I wonder if such a constant should be static and avoid the ternary.

Reply:
> @arboleya should we have an issue for collating all the little chores for mainnet?
(such as adding network URL, chain ID, etc)

@petertonysmith94 Like an epic? Definitely.